python/mxnet/gluon/data/dataloader.py

```diff
@@ -214,6 +214,7 @@ def __iter__(self):
             worker.start()
             workers.append(worker)
 
+        idx = -1
```

Comment:
?

Reply:
idx might be reference before assignment when i run pylint